dcpy/extract/ingest.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from pydantic import BaseModel
+
+from dcpy.connectors.edm import recipes
+from . import PARQUET_PATH
+
+
+class ImportDefinition(BaseModel):
```

Comment:
Rationale for this being it's nice to have a class that is strictly a definition (with optional types) separate from the actual "computed" config (which can then have non-optional types, maybe slightly different structure)